src/styles/index.scss

```diff
@@ -1,3 +1,92 @@
-body {
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
It is better to set `box-sizing: border-box;` globally to apply it to all elements. Consider adding `*, *::before, *::after { box-sizing: border-box; }` at the beginning of your stylesheet.